IPython/extensions/autoreload.py

```diff
@@ -595,15 +618,41 @@ def aimport(self, parameter_s="", stream=None):
             else:
                 stream.write("Modules to reload:\n%s\n" % " ".join(to_reload))
             stream.write("\nModules to skip:\n%s\n" % " ".join(to_skip))
-        elif modname.startswith("-"):
-            modname = modname[1:]
-            self._reloader.mark_module_skipped(modname)
         else:
             for _module in [_.strip() for _ in modname.split(",")]:
-                top_module, top_name = self._reloader.aimport_module(_module)
+                if _module.startswith("-"):
+                    _module = _module[1:].strip()
+                    self._reloader.mark_module_skipped(_module)
+                else:
+                    top_module, top_name = self._reloader.aimport_module(_module)
+
+                    # Inject module to user namespace
+                    self.shell.push({top_name: top_module})
+
+    @line_magic
+    def averbose(self, parameter_s=""):
+        r"""%averbose => Turn verbosity on/off for autoreloading.
 
-                # Inject module to user namespace
-                self.shell.push({top_name: top_module})
+        %averbose 0 or %averbose off
```

Comment:
I would avoid adding more magic numbers. off/on/log is fine.

Reply:
I'll get rid of `%averbose` as discussed above.